lib/pbench/agent/tool_meister.py

```diff
@@ -246,14 +246,14 @@ class TransientTool(Tool):
 
     _tool_type = "Transient"
 
-    def __init__(self, name, tool_opts, **kwargs):
+    def __init__(self, name: str, tool_opts: str, **kwargs):
         super().__init__(name, tool_opts, **kwargs)
         self.tool_script = f"{self.pbench_install_dir}/tool-scripts/{self.name}"
         self.start_process = None
         self.stop_process = None
         self.tool_dir = None
 
-    def install(self):
+    def install(self) -> (int, str):
```

Comment:
I'm not sure whether Python will take the *actual* tuple as a type hint. It can be picky about that, although I believe that's improving through 3.7 and 3.9. For 3.6, at least, I believe the wisdom is that it's safest to stick with `typing` names for any collection, which would make this `Tuple[int, str]`.

(FYI, I had unit tests fail for me today while I was demonstrating to Siddardh because some fixtures used `-> dict[str,str]` and Python suddenly really, really wanted them to be `-> Dict[str,str]`. I was shocked we even had anything in our toolchain that cared about type hints, but at least in some contexts we seem to be pulling in something that does.)